spec/support/books_contexts.rb

```diff
@@ -94,3 +94,68 @@
     index.wait_for_task(response['uid'])
   end
 end
+
+RSpec.shared_context 'search books with nested fields' do
+  let(:index) { client.index('books') }
+  let(:documents) do
```

Comment:
It's probably a stupid question but how is it known to apply this specific context to your tests? It did seems to be called anywhere

Reply:
Because of this line `include_context 'search books with nested fields'` in the file `spec/meilisearch/index/search/nested_fields_spec.rb`. It will include all the shared context code inside of my test file :D